phpunit/phpunit/4.7/tests/bootstrap.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+require __DIR__.'/../vendor/autoload.php';
+
+use Symfony\Component\Dotenv\Dotenv;
+
+/*
+ * Environment variables can also be specified in phpunit.xml.dist.
+ * Those variables will override any defined in .env.
+ */
+
+if (!isset($_SERVER['APP_ENV']) || 'prod' !== $_SERVER['APP_ENV']) {
```

Comment:
Why don't we use `getenv('APP_ENV')` instead of `$_SERVER['APP_ENV']`. Are there any reasons behind? Symfony recommends to use `getenv()` in docs but has `$_SERVER[]` in code which is not consistent as for me.

Reply:
See https://github.com/symfony/recipes/pull/156 about that. It should probably be updated in the docs then.

Reply:
`getenv` is an issue because of not being thread safe (probably less an issue in the PHPUnit bootstrap as this is used only in the CLI)